### PR TITLE
Add deprecation message to observability section

### PR DESCRIPTION
**Pull request by nak3**
The observability tools are deprecated but some users did notice it because monitoring section does not mention about the deprecation. https://github.com/knative/serving/issues/9596.

Also, this patch adds a minor fix which creates `knative-monitoring` namespace beforehand. https://github.com/knative/serving/issues/9606

Fixes https://github.com/knative/serving/issues/9596 https://github.com/knative/serving/issues/9606

/assign @evankanderson @mattmoor @abrennan89

### CODE REVIEW

**Review by mattmoor — COMMENTED** [review]
/lgtm

File: docs/serving/installing-logging-metrics-traces.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ weight: 40
 type: "docs"
 ---
 
+**DEPRECATED:** The monitoring tools released by Knative community are deprecated. Knative community will stop releasing them in a coming release.
```
Comment (mattmoor) [inline docs/serving/installing-logging-metrics-traces.md:8]: Maybe this should use the feature status short code?

Comment (nak3) [reply docs/serving/installing-logging-metrics-traces.md]: Yes, sounds good. Updated.

File: docs/serving/installing-logging-metrics-traces.md [hunk 2]
```diff
@@ -15,6 +17,12 @@ can skip down to the
 If you have not yet installed any observability plugins, continue to the next
 sections to do so now.
 
+All observibility plugins require that you first install the core:
```
Comment (mpetason) [inline docs/serving/installing-logging-metrics-traces.md:20]:
```suggestion
All observability plugins require that you first install the core:
```

Comment (nak3) [reply docs/serving/installing-logging-metrics-traces.md]: Thank you! Fixed.

**Comment by abrennan89** [comment]
As per discussion in Slack, let's not add this please, and instead remove the observability plugin docs entirely. They've been deprecated since 0.14 and our docs now only go as far back as 0.15. Thanks.

**Pull request closed without merging**